.github/workflows/build-containers.yml

```diff
@@ -89,6 +91,7 @@ jobs:
           # Use jq to extract the 'include' part of the JSON
           matrix=$(echo "$matrix_json" | jq -c '.include')
           echo "::set-output name=matrix::$matrix"
+          echo "::set-output name=image_list::$image_json"
```

Comment:
I think solving it this way is going to make it pretty confusing in the future as the difference between `image_list` and `images` is that they're json vs a newline-separated list. Why create a new variable at the source when you can just adjust the way you're referencing the var at the call site?